docs/API.md

```diff
@@ -124,16 +124,16 @@ Properties:
 
 Basic data type: number
 
-This parameter is any integer that will be used by The Zonemaster Backend Worker daemon to sort the test requests from highest to lowest priority.
-This parameter will typically be used in a setup where a GUI will send requests to the Backend API and would like to get response as soon as possible while at the same time using the idle time for background batch testing.
+This parameter is any integer that will be used by The Zonemaster RPC API daemon daemon to sort the test requests from highest to lowest priority.
```

Comment:
1. "daemon daemon"
2. Is it really the RPC API daemon that sorts the requests? I thought it was the Test Agents.